examples/representation/plot_mixed_data.py

```diff
@@ -0,0 +1,177 @@
+"""
+Mixed data structure and visualization.
+======================================
+
+This example demonstrates how to build and visualize mixed datasets
+containing both scalar and functional elements using the Canadian weather
+dataset. In particular, we show how to include both a function and its
+derivative, either jointly in a vector-valued functional object, or
+separately as independent entries. We also explore how to structure and
+visualize this data using `pandas` and scikit-fda's visualization tools.
+"""
+
+# Author: Luis Hebrero Garicano
+# License: MIT
+# sphinx_gallery_thumbnail_number = 1
+
+
+# %%
+# We load the Canadian weather dataset. This dataset includes daily
+# temperature and precipitation curves for 35 weather stations in Canada,
+# along with a scalar variable: the climate zone of each station.
+
+import matplotlib.pyplot as plt
```

Comment:
Could you move the imports to the first cell where they are needed? We are trying to do that in examples, as I feel it is less confusing to users.